<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copyEngineOptions</class>
 <widget class="QWidget" name="copyEngineOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>504</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_trasnfer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>380</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Transfer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Checksum</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="rsync"/>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="doRightTransfer"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Auto start</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="autoStart">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="mkpath">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="checksum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Keep the file date</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Move the whole folder</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_rsync">
         <property name="text">
          <string notr="true">Rsync</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="moveTheWholeFolder"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Create full path if not exists</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>278</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="keepDate"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Transfer the file rights</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="ignoreBlackList">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Ignore blacklist (generate lot of problem)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_error_collision">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>380</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Error and collision</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>When folder error</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="checkBoxDestinationFolderExists"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>When file error</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBoxFileError">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Put at the end</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>When file collision</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBoxFileCollision">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite if not same size and date</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite if newer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Overwrite if older</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Rename</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Overwrite if not same size</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Overwrite if modification date differs</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboBoxFolderError">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>When folder collision</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="comboBoxFolderCollision">
         <item>
          <property name="text">
           <string notr="true">Ask</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Merge</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Skip</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Rename</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Check if destination folder exists</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="renamingRules">
         <property name="text">
          <string>Renaming rules</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>193</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Delete partially transferred files</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="deletePartiallyTransferredFiles"/>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="renameTheOriginalDestination"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Rename the original destination</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>label_12</zorder>
      <zorder>label_13</zorder>
      <zorder>label_6</zorder>
      <zorder>label_7</zorder>
      <zorder>label_8</zorder>
      <zorder>comboBoxFileError</zorder>
      <zorder>comboBoxFileCollision</zorder>
      <zorder>comboBoxFolderError</zorder>
      <zorder>comboBoxFolderCollision</zorder>
      <zorder>checkBoxDestinationFolderExists</zorder>
      <zorder>renamingRules</zorder>
      <zorder>label_20</zorder>
      <zorder>deletePartiallyTransferredFiles</zorder>
      <zorder>renameTheOriginalDestination</zorder>
      <zorder>label_22</zorder>
     </widget>
     <widget class="QWidget" name="page_performance">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>380</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Performance</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QSpinBox" name="inodeThreads">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Inode threads</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="followTheStrictOrder"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="bufferSameDevice">
         <property name="suffix">
          <string>MB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="bufferOtherDevice">
         <property name="suffix">
          <string>MB</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="buffer">
         <property name="toolTip">
          <string>Uncheck this under Windows create problem</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>OS Buffer</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>197</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Disable parallel transfer when are different devices</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Buffer for different device</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>OS FLags</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Buffer for same device</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_native_copy">
         <property name="text">
          <string>OS native copy (disable speed limitation)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="os_spec_flags">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="native_copy">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_misc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>380</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Check the disk space</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkDiskSpace"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Use this folder when destination is not set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="defaultDestinationFolder"/>
         </item>
         <item>
          <widget class="QPushButton" name="defaultDestinationFolderBrowse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>353</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="filters">
         <property name="text">
          <string>Filters</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>352</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>159</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
